cmd/foreach/foreach.go

```diff
@@ -30,23 +30,31 @@ import (
 
 var exec executor.Executor = executor.NewRealExecutor()
 
-var repoFile string = "repos.txt"
+var (
+	repoFile string = "repos.txt"
+	helpFlag bool   = false
+)
 
-func parseForeachArgs(args []string) (strippedArgs []string) {
+func parseForeachArgs(args []string) []string {
+	strippedArgs := make([]string, 0)
```

Comment:
It is safer for `strippedArgs` to be initialised as an empty array rather than leaving the zero value when being returned.